charts/4allportal/templates/samba/deployment.yaml

```diff
@@ -0,0 +1,142 @@
+              name: assets
+            - mountPath: /mnt/config
+              name: config
+            - mountPath: /var/lib/samba
+              name: samba-state
+            - mountPath: /run
+              name: run
+            - mountPath: /var/run/secrets/kubernetes.io/serviceaccount
+              name: kube-api-access-abcde
+              readOnly: true
+          {{- if $.Values.samba.livenessProbe.enabled }}
+          livenessProbe:
+            exec:
+              command:
+                - bash
+                - -c
+                - timeout 1 ls -d /mnt/assets
+                - timeout 1 ls -d /mnt/config
+            failureThreshold: {{ $.Values.samba.livenessProbe.failureThreshold }}
+            initialDelaySeconds: {{ $.Values.samba.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ $.Values.samba.livenessProbe.periodSeconds }}
+            successThreshold: {{ $.Values.samba.livenessProbe.successThreshold }}
+            timeoutSeconds: {{ $.Values.samba.livenessProbe.timeoutSeconds }}
+          {{- end }}
+          {{- if $.Values.samba.readinessProbe.enabled }}
+          readinessProbe:
+            tcpSocket:
+              port: samba
+            failureThreshold: {{ $.Values.samba.readinessProbe.failureThreshold }}
+            initialDelaySeconds: {{ $.Values.samba.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ $.Values.samba.readinessProbe.periodSeconds }}
+            successThreshold: {{ $.Values.samba.readinessProbe.successThreshold }}
+            timeoutSeconds: {{ $.Values.samba.readinessProbe.timeoutSeconds }}
+          {{- end }}
+          resources: {{- toYaml $.Values.samba.resources | nindent 12 }}
+          env:
+            - name: SAMBA_CONTAINER_ID
+              value: assets
+            - name: SAMBACC_CONFIG
+              value: /etc/container-config/config.json:/etc/container-users/users.json
+            - name: SAMBA_POD_NAME
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: metadata.name
+            - name: SAMBA_POD_NAMESPACE
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.namespace
+                  apiVersion: v1
+      volumes:
+        - name: samba-container-config
+          configMap:
+            defaultMode: 420
+            name: "{{ include "common.names.fullname" $ }}-samba"
+        - name: users-config
+          secret:
+            defaultMode: 420
+            secretName: {{ include "common.secrets.name" (dict "existingSecret" (dict) "defaultNameSuffix" "samba" "context" $) }}
+            items:
+              - key: users
+                path: users.json
+        - name: assets
+        {{- include "4allportal.fourallportal.persistence.assets.mount" $ | nindent 10 }}
+        - name: config
+        {{- include "4allportal.fourallportal.persistence.config.mount" $ | nindent 10 }}
```

Comment:
Still can't find template for this and assets.mount, or is that an existing value?

Reply:
nevermind, exists already